doc/classes/GPUParticles2D.xml

```diff
@@ -18,6 +18,16 @@
 				Returns a rectangle containing the positions of all existing particles.
 			</description>
 		</method>
+		<method name="emit_particle">
+			<return type="void" />
+			<argument index="0" name="xform" type="Transform2D" />
+			<argument index="1" name="velocity" type="Vector2" />
+			<argument index="2" name="color" type="Color" />
+			<argument index="3" name="custom" type="Color" />
+			<argument index="4" name="flags" type="int" />
+			<description>
```

Comment:
The docs should be filled in for new methods and constants

Reply:
Will fill in as soon as possible. Is it okay if I document the 3D ones too? Should I rebase as well?

Reply:
You don't need to rebase right now as there are no conflicts. If you want to document the 3D functions as well it would be very welcome!

Reply:
I filled in the docs, but could change it to make it more elaborate/concise if needed.